prov/rxm/src/rxm.h

```diff
@@ -356,6 +356,15 @@ enum rxm_proto_state {
 
 extern char *rxm_proto_state_str[];
 
+enum {
+	rxm_ctrl_eager,
+	rxm_ctrl_seg,
+	rxm_ctrl_rndv,
+	rxm_ctrl_rndv_ack,
+	rxm_ctrl_atomic,
+	rxm_ctrl_atomic_resp,
+};
```

Comment:
Do these have the same values as the common ones?  Or does this change modify the protocol?

Reply:
These have different values. I'll update the protocol version.